eng/pipelines/templates/jobs/installation.check.yml

```diff
@@ -0,0 +1,33 @@
+parameters:
+  - name: Artifact
+    type: object
+    default: {}
+  - name: ArtifactName
+    type: string
+    default: "not-specified"
+
+jobs: 
+  - job: InstallationCheck
+    displayName: "Installation Check"
+    variables:
+      - template: ../variables/globals.yml
+    steps:
+      - download: current
+        artifact: ${{parameters.ArtifactName}}-signed
+      - pwsh: |
+          Write-Host "Copy-Item $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed/${{parameters.Artifact.groupId}}/${{parameters.Artifact.name}}/" "$(Pipeline.Workspace)/install-check-dir/${{parameters.Artifact.name}} -Recurse"
+          Copy-Item "${{parameters.ArtifactName}}-signed/${{parameters.Artifact.groupId}}/${{parameters.Artifact.name}}/" "install-check-dir/${{parameters.Artifact.name}}" -Recurse
+          Write-Host "$pomf = Get-ChildItem install-check-dir/${{parameters.Artifact.name}}/*.pom"
+          $pomf = Get-ChildItem install-check-dir/${{parameters.Artifact.name}}/*.pom
+          Write-Host "Rename-Item $($pomf) pom.xml"
+          Rename-Item $pomf pom.xml
+          Copy-Item "$(System.DefaultWorkingDirectory)/eng" "install-check-dir/${{ parameters.Artifact.name }}/eng" -Recurse
+        workingDirectory: $(Pipeline.Workspace)
+        displayName: "Copy Artifacts and Directories"
+      - template: /eng/pipelines/templates/steps/install-reporting-tools.yml
+      - task: Maven@3
+        displayName: "Verify Package Installation"
+        inputs:
+          mavenPomFile: "$(Pipeline.Workspace)/install-check-dir/${{ parameters.Artifact.name }}/pom.xml"
```

Comment:
Does this only try to install the dependencies or will it also install the package itself?

Reply:
It installs the dependencies, won't install the package itself
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1065831&view=logs&j=820f9f32-a56c-5bf9-3c6e-075e0e71e80f&t=96d11280-199e-5458-d25a-c72f20aad7c2&l=210